### ISSUE TITLE
[Windows] App crashes with Lost connection to device

### ISSUE THREAD

**Issue by leamlidara** [flutter/flutter#101015]
Flutter app always lose connections when debugging on Windows (Windows App) but process is remaining open. If we are on breakpoint, the program stuck and losing connection to the debug program.

<details><summary>flutter doctor -v</summary>
<p>

```
[✓] Flutter (Channel stable, 2.10.4, on Microsoft Windows [Version 10.0.22000.593], locale en-US)
    • Flutter version 2.10.4 at D:\Program Files\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision c860cba910 (5 days ago), 2022-03-25 00:23:12 -0500
    • Engine revision 57d3bac3dd
    • Dart version 2.16.2
    • DevTools version 2.9.2

[✓] Android toolchain - develop for Android devices (Android SDK version 32.0.0)
    • Android SDK at C:\Users\leam_\AppData\Local\Android\sdk
    • Platform android-32, build-tools 32.0.0
    • Java binary at: D:\Program Files\android-studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 11.0.11+9-b60-7590822)
    • All Android licenses accepted.

[✓] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[✓] Visual Studio - develop for Windows (Visual Studio Enterprise 2019 16.7.25)
    • Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise
    • Visual Studio Enterprise 2019 version 16.7.32125.265
    • Windows 10 SDK version 10.0.18362.0

[✓] Android Studio (version 2021.1)
    • Android Studio at D:\Program Files\android-studio
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.11+9-b60-7590822)

[✓] VS Code (version 1.65.2)
    • VS Code at C:\Users\leam_\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.36.0

[✓] Connected device (3 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.22000.593]
    • Chrome (web)      • chrome  • web-javascript • Google Chrome 99.0.4844.82
    • Edge (web)        • edge    • web-javascript • Microsoft Edge 99.0.1150.46

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```
</p>
</details>

<details><summary>flutter run -v</summary>
<p>

```
[  +70 ms] executing: [D:\Program Files\flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[ +483 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[   +2 ms] c860cba910319332564e1e9d470a17074c1f2dfd
[   +1 ms] executing: [D:\Program Files\flutter/] git tag --points-at c860cba910319332564e1e9d470a17074c1f2dfd
[  +47 ms] Exit code 0 from: git tag --points-at c860cba910319332564e1e9d470a17074c1f2dfd
[        ] 2.10.4
[  +13 ms] executing: [D:\Program Files\flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[  +45 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[        ] origin/stable
[        ] executing: [D:\Program Files\flutter/] git ls-remote --get-url origin
[  +37 ms] Exit code 0 from: git ls-remote --get-url origin
[        ] https://github.com/flutter/flutter.git
[ +182 ms] executing: [D:\Program Files\flutter/] git rev-parse --abbrev-ref HEAD
[  +35 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[        ] stable
[  +66 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +3 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'WindowsUwpEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[   +5 ms] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +66 ms] executing: C:\Users\leam_\AppData\Local\Android\sdk\platform-tools\adb.exe devices -l
[  +48 ms] List of devices attached
[   +8 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[  +11 ms] Artifact Instance of 'WindowsUwpEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[ +114 ms] Multiple devices found:
[  +25 ms] Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.22000.593]
[   +2 ms] Chrome (web)      • chrome  • web-javascript • Google Chrome 99.0.4844.82
[   +1 ms] Edge (web)        • edge    • web-javascript • Microsoft Edge 99.0.1150.46
[   +1 ms] [1]: Windows (windows)
[        ] [2]: Chrome (chrome)
[        ] [3]: Edge (edge)
[   +2 ms] Please choose one (To quit, press "q/Q")
[        ] :
[+6457 ms] 1
[ +123 ms] Skipping pub get: version match.
[  +75 ms] Found plugin device_info at D:\Program Files\flutter\.pub-cache\hosted\pub.dartlang.org\device_info-2.0.3\
[  +21 ms] Found plugin device_info_plus at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\device_info_plus-3.2.2\
[   +7 ms] Found plugin device_info_plus_linux at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\device_info_plus_linux-2.1.1\
[   +4 ms] Found plugin device_info_plus_macos at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\device_info_plus_macos-2.2.2\
[   +6 ms] Found plugin device_info_plus_web at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\device_info_plus_web-2.1.0\
[   +2 ms] Found plugin device_info_plus_windows at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\device_info_plus_windows-2.1.1\
[  +11 ms] Found plugin flutter_acrylic at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\flutter_acrylic-1.0.0+2\
[  +10 ms] Found plugin platform_device_id at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\platform_device_id-1.0.1\
[   +5 ms] Found plugin platform_device_id_linux at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\platform_device_id_linux-1.0.0\
[   +2 ms] Found plugin platform_device_id_macos at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\platform_device_id_macos-1.0.0\
[   +3 ms] Found plugin platform_device_id_web at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\platform_device_id_web-1.0.0\
[   +1 ms] Found plugin platform_device_id_windows at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\platform_device_id_windows-1.0.0\
[  +15 ms] Found plugin url_launcher at D:\Program Files\flutter\.pub-cache\hosted\pub.dartlang.org\url_launcher-6.0.20\[   +2 ms] Found plugin url_launcher_android at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\url_launcher_android-6.0.15\
[   +1 ms] Found plugin url_launcher_ios at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\url_launcher_ios-6.0.15\
[   +2 ms] Found plugin url_launcher_linux at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\url_launcher_linux-3.0.0\
[   +3 ms] Found plugin url_launcher_macos at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\url_launcher_macos-3.0.0\
[   +2 ms] Found plugin url_launcher_web at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\url_launcher_web-2.0.9\
[   +2 ms] Found plugin url_launcher_windows at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\url_launcher_windows-3.0.0\
[   +3 ms] Found plugin window_manager at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\window_manager-0.1.9\
[  +62 ms] Found plugin device_info at D:\Program Files\flutter\.pub-cache\hosted\pub.dartlang.org\device_info-2.0.3\
[   +3 ms] Found plugin device_info_plus at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\device_info_plus-3.2.2\
[   +2 ms] Found plugin device_info_plus_linux at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\device_info_plus_linux-2.1.1\
[   +1 ms] Found plugin device_info_plus_macos at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\device_info_plus_macos-2.2.2\
[   +4 ms] Found plugin device_info_plus_web at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\device_info_plus_web-2.1.0\
[   +1 ms] Found plugin device_info_plus_windows at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\device_info_plus_windows-2.1.1\
[   +4 ms] Found plugin flutter_acrylic at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\flutter_acrylic-1.0.0+2\
[   +6 ms] Found plugin platform_device_id at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\platform_device_id-1.0.1\
[   +2 ms] Found plugin platform_device_id_linux at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\platform_device_id_linux-1.0.0\
[   +2 ms] Found plugin platform_device_id_macos at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\platform_device_id_macos-1.0.0\
[   +2 ms] Found plugin platform_device_id_web at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\platform_device_id_web-1.0.0\
[   +1 ms] Found plugin platform_device_id_windows at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\platform_device_id_windows-1.0.0\
[   +7 ms] Found plugin url_launcher at D:\Program Files\flutter\.pub-cache\hosted\pub.dartlang.org\url_launcher-6.0.20\[   +2 ms] Found plugin url_launcher_android at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\url_launcher_android-6.0.15\
[   +2 ms] Found plugin url_launcher_ios at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\url_launcher_ios-6.0.15\
[   +2 ms] Found plugin url_launcher_linux at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\url_launcher_linux-3.0.0\
[   +3 ms] Found plugin url_launcher_macos at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\url_launcher_macos-3.0.0\
[   +2 ms] Found plugin url_launcher_web at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\url_launcher_web-2.0.9\
[   +1 ms] Found plugin url_launcher_windows at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\url_launcher_windows-3.0.0\
[   +3 ms] Found plugin window_manager at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\window_manager-0.1.9\
[  +16 ms] Generating D:\Projects\client1\android\app\src\main\java\io\flutter\plugins\GeneratedPluginRegistrant.java
[ +121 ms] Initializing file store
[  +13 ms] Skipping target: gen_localizations
[   +7 ms] gen_dart_plugin_registrant: Starting due to {InvalidatedReasonKind.inputChanged: The following inputs have
updated contents: D:\Projects\client1\.dart_tool\package_config_subset,D:\Projects\client1\.dart_tool\flutter_build\generated_main.dart}
[  +22 ms] Found plugin device_info at D:\Program Files\flutter\.pub-cache\hosted\pub.dartlang.org\device_info-2.0.3\
[   +2 ms] Found plugin device_info_plus at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\device_info_plus-3.2.2\
[   +3 ms] Found plugin device_info_plus_linux at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\device_info_plus_linux-2.1.1\
[   +2 ms] Found plugin device_info_plus_macos at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\device_info_plus_macos-2.2.2\
[   +2 ms] Found plugin device_info_plus_web at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\device_info_plus_web-2.1.0\
[   +1 ms] Found plugin device_info_plus_windows at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\device_info_plus_windows-2.1.1\
[   +4 ms] Found plugin flutter_acrylic at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\flutter_acrylic-1.0.0+2\
[   +8 ms] Found plugin platform_device_id at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\platform_device_id-1.0.1\
[   +3 ms] Found plugin platform_device_id_linux at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\platform_device_id_linux-1.0.0\
[   +1 ms] Found plugin platform_device_id_macos at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\platform_device_id_macos-1.0.0\
[   +1 ms] Found plugin platform_device_id_web at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\platform_device_id_web-1.0.0\
[   +1 ms] Found plugin platform_device_id_windows at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\platform_device_id_windows-1.0.0\
[  +11 ms] Found plugin url_launcher at D:\Program Files\flutter\.pub-cache\hosted\pub.dartlang.org\url_launcher-6.0.20\[   +1 ms] Found plugin url_launcher_android at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\url_launcher_android-6.0.15\
[   +1 ms] Found plugin url_launcher_ios at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\url_launcher_ios-6.0.15\
[   +1 ms] Found plugin url_launcher_linux at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\url_launcher_linux-3.0.0\
[   +1 ms] Found plugin url_launcher_macos at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\url_launcher_macos-3.0.0\
[   +1 ms] Found plugin url_launcher_web at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\url_launcher_web-2.0.9\
[   +1 ms] Found plugin url_launcher_windows at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\url_launcher_windows-3.0.0\
[   +7 ms] Found plugin window_manager at D:\Program
Files\flutter\.pub-cache\hosted\pub.dartlang.org\window_manager-0.1.9\
[  +12 ms] gen_dart_plugin_registrant: Complete
[   +4 ms] Skipping target: _composite
[   +3 ms] complete
[   +7 ms] Launching lib\main.dart on Windows in debug mode...
[   +7 ms] D:\Program Files\flutter\bin\cache\dart-sdk\bin\dart.exe --disable-dart-dev D:\Program
Files\flutter\bin\cache\artifacts\engine\windows-x64\frontend_server.dart.snapshot --sdk-root D:\Program
Files\flutter\bin\cache\artifacts\engine\common\flutter_patched_sdk/ --incremental --target=flutter
--debugger-module-names --experimental-emit-debug-metadata -DFLUTTER_WEB_AUTO_DETECT=true --output-dill
C:\Users\leam_\AppData\Local\Temp\flutter_tools.57b4ef2b\flutter_tool.5ebf67a2\app.dill --packages D:\Projects\client1\.dart_tool\package_config.json -Ddart.vm.profile=false -Ddart.vm.product=false --enable-asserts
--track-widget-creation --filesystem-scheme org-dartlang-root --initialize-from-dill
build\c075001b96339384a97db4862b8ab8db.cache.dill.track.dill --enable-experiment=alternative-invalidation-strategy
[  +32 ms] executing: C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe -format json -products *
-utf8 -latest -version 16 -requires Microsoft.VisualStudio.Workload.NativeDesktop
Microsoft.VisualStudio.Component.VC.Tools.x86.x64 Microsoft.VisualStudio.Component.VC.CMake.Project
[  +33 ms] Exit code 0 from: C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe -format json -products* -utf8 -latest -version 16 -requires Microsoft.VisualStudio.Workload.NativeDesktop
Microsoft.VisualStudio.Component.VC.Tools.x86.x64 Microsoft.VisualStudio.Component.VC.CMake.Project
[        ] [
             {
               "instanceId": "77eb8650",
               "installDate": "2022-03-08T15:23:11Z",
               "installationName": "VisualStudio/16.7.25+32125.265",
               "installationPath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Enterprise",
               "installationVersion": "16.7.32125.265",
               "productId": "Microsoft.VisualStudio.Product.Enterprise",
               "productPath": "C:\\Program Files (x86)\\Microsoft Visual
               Studio\\2019\\Enterprise\\Common7\\IDE\\devenv.exe",
               "state": 4294967295,
               "isComplete": true,
               "isLaunchable": true,
               "isPrerelease": false,
               "isRebootRequired": false,
               "displayName": "Visual Studio Enterprise 2019",
               "description": "Scalable, end-to-end solution for teams of any size",
               "channelId": "VisualStudio.16.Release",
               "channelUri": "https://aka.ms/vs/16/release/channel",
               "enginePath": "C:\\Program Files (x86)\\Microsoft Visual
               Studio\\Installer\\resources\\app\\ServiceHub\\Services\\Microsoft.VisualStudio.Setup.Service",
               "installChannelUri": "https://aka.ms/vs/16/release/720956465_-938549663/channel",
               "installedChannelUri": "https://aka.ms/vs/16/release/channel",
               "releaseNotes": "https://go.microsoft.com/fwlink/?LinkId=660893#16.7.25",
               "thirdPartyNotices": "https://go.microsoft.com/fwlink/?LinkId=660909",
               "updateDate": "2022-03-08T15:23:11.9270992Z",
               "catalog": {
                 "buildBranch": "d16.7",
                 "buildVersion": "16.7.32125.265",
                 "id": "VisualStudio/16.7.25+32125.265",
                 "localBuild": "build-lab",
                 "manifestName": "VisualStudio",
                 "manifestType": "installer",
                 "productDisplayVersion": "16.7.25",
                 "productLine": "Dev16",
                 "productLineVersion": "2019",
                 "productMilestone": "RTW",
                 "productMilestoneIsPreRelease": "False",
                 "productName": "Visual Studio",
                 "productPatchVersion": "25",
                 "productPreReleaseMilestoneSuffix": "1.0",
                 "productSemanticVersion": "16.7.25+32125.265",
                 "requiredEngineVersion": "2.7.3136.13130"
               },
               "properties": {
                 "campaignId": "2030",
                 "channelManifestId": "VisualStudio.16.Release/16.7.25+32125.265",
                 "nickname": "",
                 "setupEngineFilePath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\Installer\\setup.exe"
               }
             }
           ]
[   +4 ms] Building Windows application...
[  +29 ms] <- compile file:///D:/Projects/client1/.dart_tool/flutter_build/generated_main.dart
[   +2 ms] executing: C:\Program Files (x86)\Microsoft Visual
Studio\2019\Enterprise\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe -S D:\Projects\client1\windows -B build\windows -G Visual Studio 16 2019
[  +77 ms] -- Selecting Windows SDK version 10.0.18362.0 to target Windows 10.0.22000.
[  +28 ms] -- Configuring done
[ +354 ms] -- Generating done
[  +35 ms] -- Build files have been written to: D:/Projects/client1/build/windows
[   +9 ms] executing: C:\Program Files (x86)\Microsoft Visual
Studio\2019\Enterprise\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe --build build\windows --config
Debug --target INSTALL --verbose
[ +181 ms] Microsoft (R) Build Engine version 16.7.2+b60ddb6f4 for .NET Framework
[        ] Copyright (C) Microsoft Corporation. All rights reserved.
[  +52 ms] Build started 3/30/2022 11:18:34 AM.
[ +256 ms] Project "D:\Projects\client1\build\windows\INSTALL.vcxproj" on node 1 (default targets).
[        ] Project "D:\Projects\client1\build\windows\INSTALL.vcxproj" (1) is building "D:\Projects\client1\build\windows\ZERO_CHECK.vcxproj" (2) on node 1 (default targets).
[   +1 ms] InitializeBuildStatus:
[        ]   Creating "x64\Debug\ZERO_CHECK\ZERO_CHECK.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[  +29 ms] CustomBuild:
[        ]   All outputs are up-to-date.
[  +25 ms] FinalizeBuildStatus:
[        ]   Deleting file "x64\Debug\ZERO_CHECK\ZERO_CHECK.tlog\unsuccessfulbuild".
[   +2 ms]   Touching "x64\Debug\ZERO_CHECK\ZERO_CHECK.tlog\ZERO_CHECK.lastbuildstate".
[   +2 ms] Done Building Project "D:\Projects\client1\build\windows\ZERO_CHECK.vcxproj" (default targets).
[ +136 ms] Project "D:\Projects\client1\build\windows\INSTALL.vcxproj" (1) is building "D:\Projects\client1\build\windows\ALL_BUILD.vcxproj" (3) on node 1 (default targets).
[        ] Project "D:\Projects\client1\build\windows\ALL_BUILD.vcxproj" (3) is building
"D:\Projects\client1\build\windows\runner\client1.vcxproj" (4) on node 1 (default targets).
[        ] Project "D:\Projects\client1\build\windows\runner\client1.vcxproj" (4) is building
"D:\Projects\client1\build\windows\flutter\flutter_assemble.vcxproj" (5) on node 1 (default targets).
[   +1 ms] InitializeBuildStatus:
[        ]   Creating "x64\Debug\flutter_assemble\flutter_assemble.tlog\unsuccessfulbuild" because "AlwaysCreate" was
specified.
[   +3 ms] CustomBuild:
[        ]   "The build of 'D:\Projects\client1\build\windows\CMakeFiles\ddc891142abc3f1c3e475671d5540e7d\flutter_assemble.rule' depends on
'D:\PROJECTS\CLIENT1\WINDOWS\FLUTTER\EPHEMERAL\FLUTTER_WINDOWS.DLL' which is produced by the build of
'D:\Projects\client1\build\windows\CMakeFiles\e334fad2b5f69d92280d12f40a5090d5\flutter_windows.dll.rule'.
The items cannot be built in parallel."
[  +76 ms]   Generating D:/Projects/client1/windows/flutter/ephemeral/flutter_windows.dll,
D:/Projects/client1/windows/flutter/ephemeral/flutter_export.h, D:/Projects/client1/windows/flutter/ephemeral/flutter_windows.h, D:/Projects/client1/windows/flutter/ephemeral/flutter_messenger.h, D:/Projects/client1/windows/flutter/ephemeral/flutter_plugin_registrar.h, D:/Projects/client1/windows/flutter/ephemeral/flutter_texture_registrar.h, D:/Projects/client1/windows/flutter/ephemeral/cpp_client_wrapper/core_implementations.cc, D:/Projects/client1/windows/flutter/ephemeral/cpp_client_wrapper/standard_codec.cc, D:/Projects/client1/windows/flutter/ephemeral/cpp_client_wrapper/plugin_registrar.cc, D:/Projects/client1/windows/flutter/ephemeral/cpp_client_wrapper/flutter_engine.cc, D:/Projects/client1/windows/flutter/ephemeral/cpp_client_wrapper/flutter_view_controller.cc, _phony_
[+1690 ms]   [  +82 ms] executing: [D:\Program Files\flutter/] git -c log.showSignature=false log -n 1
--pretty=format:%H
[ +538 ms]   [ +592 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[        ]   [        ] c860cba910319332564e1e9d470a17074c1f2dfd
[        ]   [        ] executing: [D:\Program Files\flutter/] git tag --points-at
c860cba910319332564e1e9d470a17074c1f2dfd
[  +94 ms]   [  +98 ms] Exit code 0 from: git tag --points-at c860cba910319332564e1e9d470a17074c1f2dfd
[   +1 ms]   [        ] 2.10.4
[  +28 ms]   [  +28 ms] executing: [D:\Program Files\flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[  +46 ms]   [  +47 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[        ]   [        ] origin/stable
[        ]   [        ] executing: [D:\Program Files\flutter/] git ls-remote --get-url origin
[  +58 ms]   [  +58 ms] Exit code 0 from: git ls-remote --get-url origin
[   +1 ms]   [        ] https://github.com/flutter/flutter.git
[ +128 ms]   [ +129 ms] executing: [D:\Program Files\flutter/] git rev-parse --abbrev-ref HEAD
[  +61 ms]   [  +62 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[        ]   [        ] stable
[ +107 ms]   [ +107 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +3 ms]   [   +8 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'WindowsUwpEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[ +179 ms]   [ +181 ms] Artifact Instance of 'MaterialFonts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'GradleWrapper' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'FlutterSdk' is not required, skipping update.
[   +7 ms]   [  +11 ms] Artifact Instance of 'WindowsUwpEngineArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[   +2 ms]   [        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FontSubsetArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'PubDependencies' is not required, skipping update.
[ +151 ms]   [ +160 ms] Initializing file store
[  +58 ms]   [  +57 ms] Done initializing file store
[ +156 ms]   [ +157 ms] Skipping target: gen_localizations
[  +17 ms]   [  +17 ms] Skipping target: gen_dart_plugin_registrant
[+3216 ms]   [+3216 ms] Skipping target: unpack_windows
[ +780 ms]   [ +780 ms] kernel_snapshot: Starting due to {InvalidatedReasonKind.inputChanged: The following inputs have
updated contents: D:\Projects\client1\lib\screen\controls\dialog.dart,D:\Projects\client1\lib\screen\authentication\verifyRegister.dart}
[  +17 ms]   [  +16 ms] D:\Program Files\flutter\bin\cache\dart-sdk\bin\dart.exe --disable-dart-dev D:\Program
Files\flutter\bin\cache\artifacts\engine\windows-x64\frontend_server.dart.snapshot --sdk-root D:\Program
Files\flutter\bin\cache\artifacts\engine\common\flutter_patched_sdk/ --target=flutter
--no-print-incremental-dependencies -DFLUTTER_WEB_AUTO_DETECT=true -Ddart.vm.profile=false -Ddart.vm.product=false
--enable-asserts --track-widget-creation --packages D:\Projects\client1\.dart_tool\package_config.json
--output-dill D:\Projects\client1\.dart_tool\flutter_build\44dc95562a8cc4eca6a191a6953ace4b\app.dill
--depfile D:\Projects\client1\.dart_tool\flutter_build\44dc95562a8cc4eca6a191a6953ace4b\kernel_snapshot.d
D:\Projects\client1\.dart_tool\flutter_build\generated_main.dart
[+16238 ms]   [+16239 ms] kernel_snapshot: Complete
[ +346 ms]   [ +347 ms] invalidated build due to missing files: D:\Projects\client1\DOES_NOT_EXIST_RERUN_FOR_WILDCARD882001607
[ +372 ms]   [ +372 ms] debug_bundle_windows_assets: Starting due to {InvalidatedReasonKind.inputChanged: The following
inputs have updated contents: D:\Projects\client1\.dart_tool\flutter_build\44dc95562a8cc4eca6a191a6953ace4b\app.dill,D:\Projects\client1\assets\i18n\en.json,D:\Projects\client1\build\flutter_assets\kernel_blob.bin,D:\Projects\client1\build\flutter_assets\assets/i18n/en.json, InvalidatedReasonKind.inputMissing: The following inputs were
missing: D:\Projects\client1\DOES_NOT_EXIST_RERUN_FOR_WILDCARD882001607}
[ +232 ms]   [ +232 ms] Manifest contained wildcard assets. Inserting missing file into build graph to force rerun. for
more information see #56466.
[  +89 ms]   [  +88 ms] debug_bundle_windows_assets: Complete
[ +409 ms]   [ +410 ms] Persisting file store
[  +16 ms]   [  +16 ms] Done persisting file store
[  +10 ms]   [  +10 ms] build succeeded.
[  +11 ms]   [  +10 ms] "flutter assemble" took 22,415ms.
[ +231 ms]   [ +231 ms] ensureAnalyticsSent: 223ms
[   +4 ms]   [   +4 ms] Running shutdown hooks
[   +1 ms]   [   +1 ms] Shutdown hooks complete
[   +1 ms]   [   +1 ms] exiting with code 0
[ +127 ms] FinalizeBuildStatus:
[   +1 ms]   Deleting file "x64\Debug\flutter_assemble\flutter_assemble.tlog\unsuccessfulbuild".
[   +1 ms]   Touching "x64\Debug\flutter_assemble\flutter_assemble.tlog\flutter_assemble.lastbuildstate".
[   +1 ms] Done Building Project "D:\Projects\client1\build\windows\flutter\flutter_assemble.vcxproj"
(default targets).
[  +99 ms] Project "D:\Projects\client1\build\windows\runner\client1.vcxproj" (4) is building
"D:\Projects\client1\build\windows\plugins\flutter_acrylic\flutter_acrylic_plugin.vcxproj" (6) on node 1
(default targets).
[   +1 ms] Project "D:\Projects\client1\build\windows\plugins\flutter_acrylic\flutter_acrylic_plugin.vcxproj" (6) is building "D:\Projects\client1\build\windows\flutter\flutter_wrapper_plugin.vcxproj" (7) on node 1 (default targets).
[   +1 ms] InitializeBuildStatus:
[        ]   Creating "flutter_wrapper_plugin.dir\Debug\flutter_.84EE2A2C.tlog\unsuccessfulbuild" because "AlwaysCreate"
was specified.
[   +1 ms] CustomBuild:
[        ]   All outputs are up-to-date.
[  +72 ms] ClCompile:
[        ]   All outputs are up-to-date.
[  +19 ms] Lib:
[        ]   All outputs are up-to-date.
[        ]   flutter_wrapper_plugin.vcxproj -> D:\Projects\client1\build\windows\flutter\Debug\flutter_wrapper_plugin.lib
[  +12 ms] FinalizeBuildStatus:
[   +1 ms]   Deleting file "flutter_wrapper_plugin.dir\Debug\flutter_.84EE2A2C.tlog\unsuccessfulbuild".
[   +1 ms]   Touching "flutter_wrapper_plugin.dir\Debug\flutter_.84EE2A2C.tlog\flutter_wrapper_plugin.lastbuildstate".
[        ] Done Building Project "D:\Projects\client1\build\windows\flutter\flutter_wrapper_plugin.vcxproj"
(default targets).
[   +5 ms] InitializeBuildStatus:
[        ]   Creating "flutter_acrylic_plugin.dir\Debug\flutter_.6BF0A343.tlog\unsuccessfulbuild" because "AlwaysCreate"
was specified.
[        ] CustomBuild:
[        ]   All outputs are up-to-date.
[   +4 ms] ClCompile:
[        ]   All outputs are up-to-date.
[  +29 ms] Link:
[   +1 ms]   All outputs are up-to-date.
[   +2 ms]   flutter_acrylic_plugin.vcxproj -> D:\Projects\client1\build\windows\plugins\flutter_acrylic\Debug\flutter_acrylic_plugin.dll
[  +28 ms] FinalizeBuildStatus:
[   +1 ms]   Deleting file "flutter_acrylic_plugin.dir\Debug\flutter_.6BF0A343.tlog\unsuccessfulbuild".
[        ]   Touching "flutter_acrylic_plugin.dir\Debug\flutter_.6BF0A343.tlog\flutter_acrylic_plugin.lastbuildstate".
[        ] Done Building Project "D:\Projects\client1\build\windows\plugins\flutter_acrylic\flutter_acrylic_plugin.vcxproj" (default targets).
[  +68 ms] Project "D:\Projects\client1\build\windows\runner\client1.vcxproj" (4) is building
"D:\Projects\client1\build\windows\flutter\flutter_wrapper_app.vcxproj" (8) on node 1 (default targets).
[   +1 ms] InitializeBuildStatus:
[        ]   Creating "flutter_wrapper_app.dir\Debug\flutter_.6104C123.tlog\unsuccessfulbuild" because "AlwaysCreate"
was specified.
[   +1 ms] CustomBuild:
[        ]   All outputs are up-to-date.
[   +9 ms] ClCompile:
[        ]   All outputs are up-to-date.
[   +9 ms] Lib:
[        ]   All outputs are up-to-date.
[        ]   flutter_wrapper_app.vcxproj -> D:\Projects\client1\build\windows\flutter\Debug\flutter_wrapper_app.lib
[   +8 ms] FinalizeBuildStatus:
[        ]   Deleting file "flutter_wrapper_app.dir\Debug\flutter_.6104C123.tlog\unsuccessfulbuild".
[   +7 ms]   Touching "flutter_wrapper_app.dir\Debug\flutter_.6104C123.tlog\flutter_wrapper_app.lastbuildstate".
[        ] Done Building Project "D:\Projects\client1\build\windows\flutter\flutter_wrapper_app.vcxproj"
(default targets).
[  +34 ms] Project "D:\Projects\client1\build\windows\runner\client1.vcxproj" (4) is building
"D:\Projects\client1\build\windows\plugins\platform_device_id_windows\platform_device_id_windows_plugin.vcxproj" (9) on node 1
(default targets).
[        ] InitializeBuildStatus:
[        ]   Creating "platform_device_id_windows_plugin.dir\Debug\platform.5EB403D0.tlog\unsuccessfulbuild" because
"AlwaysCreate" was specified.
[        ] CustomBuild:
[        ]   All outputs are up-to-date.
[   +3 ms] ClCompile:
[        ]   All outputs are up-to-date.
[   +8 ms] Link:
[        ]   All outputs are up-to-date.
[        ]   platform_device_id_windows_plugin.vcxproj -> D:\Projects\client1\build\windows\plugins\platform_device_id_windows\Debug\platform_device_id_windows_plugin.dll
[   +4 ms] FinalizeBuildStatus:
[        ]   Deleting file "platform_device_id_windows_plugin.dir\Debug\platform.5EB403D0.tlog\unsuccessfulbuild".
[        ]   Touching
"platform_device_id_windows_plugin.dir\Debug\platform.5EB403D0.tlog\platform_device_id_windows_plugin.lastbuildstate".
[        ] Done Building Project "D:\Projects\client1\build\windows\plugins\platform_device_id_windows\platform_device_id_windows_plugin.vcxproj" (default
targets).
[  +88 ms] Project "D:\Projects\client1\build\windows\runner\client1.vcxproj" (4) is building
"D:\Projects\client1\build\windows\plugins\url_launcher_windows\url_launcher_windows_plugin.vcxproj" (10)
on node 1 (default targets).
[   +1 ms] InitializeBuildStatus:
[        ]   Creating "url_launcher_windows_plugin.dir\Debug\url_laun.84A9564A.tlog\unsuccessfulbuild" because
"AlwaysCreate" was specified.
[   +1 ms] CustomBuild:
[        ]   All outputs are up-to-date.
[  +10 ms] ClCompile:
[   +2 ms]   All outputs are up-to-date.
[   +9 ms] Link:
[        ]   All outputs are up-to-date.
[        ]   url_launcher_windows_plugin.vcxproj -> D:\Projects\client1\build\windows\plugins\url_launcher_windows\Debug\url_launcher_windows_plugin.dll
[  +29 ms] FinalizeBuildStatus:
[   +1 ms]   Deleting file "url_launcher_windows_plugin.dir\Debug\url_laun.84A9564A.tlog\unsuccessfulbuild".
[   +1 ms]   Touching
"url_launcher_windows_plugin.dir\Debug\url_laun.84A9564A.tlog\url_launcher_windows_plugin.lastbuildstate".
[   +1 ms] Done Building Project "D:\Projects\client1\build\windows\plugins\url_launcher_windows\url_launcher_windows_plugin.vcxproj" (default targets).
[  +64 ms] Project "D:\Projects\client1\build\windows\runner\client1.vcxproj" (4) is building
"D:\Projects\client1\build\windows\plugins\window_manager\window_manager_plugin.vcxproj" (11) on node 1
(default targets).
[   +2 ms] InitializeBuildStatus:
[        ]   Creating "window_manager_plugin.dir\Debug\window_m.A065D93B.tlog\unsuccessfulbuild" because "AlwaysCreate"
was specified.
[   +2 ms] CustomBuild:
[   +1 ms]   All outputs are up-to-date.
[  +16 ms] ClCompile:
[        ]   All outputs are up-to-date.
[  +19 ms] Link:
[   +1 ms]   All outputs are up-to-date.
[   +1 ms]   window_manager_plugin.vcxproj -> D:\Projects\client1\build\windows\plugins\window_manager\Debug\window_manager_plugin.dll
[  +29 ms] FinalizeBuildStatus:
[        ]   Deleting file "window_manager_plugin.dir\Debug\window_m.A065D93B.tlog\unsuccessfulbuild".
[   +1 ms]   Touching "window_manager_plugin.dir\Debug\window_m.A065D93B.tlog\window_manager_plugin.lastbuildstate".
[   +1 ms] Done Building Project "D:\Projects\client1\build\windows\plugins\window_manager\window_manager_plugin.vcxproj" (default targets).
[  +22 ms] InitializeBuildStatus:
[   +1 ms]   Creating "client1.dir\Debug\client1.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[   +4 ms] CustomBuild:
[   +1 ms]   All outputs are up-to-date.
[  +28 ms] ClCompile:
[   +1 ms]   C:\Program Files (x86)\Microsoft Visual
Studio\2019\Enterprise\VC\Tools\MSVC\14.27.29110\bin\HostX64\x64\CL.exe /c /I"D:\Projects\client1\windows"
/I"D:\Projects\client1\windows\flutter\ephemeral" /I"D:\Projects\client1\windows\flutter\ephemeral\cpp_client_wrapper\include" /I"D:\Projects\client1\windows\flutter\ephemeral\.plugin_symlinks\flutter_acrylic\windows\include" /I"D:\Projects\client1\windows\flutter\ephemeral\.plugin_symlinks\platform_device_id_windows\windows\include"
/I"D:\Projects\client1\windows\flutter\ephemeral\.plugin_symlinks\url_launcher_windows\windows\include"
/I"D:\Projects\client1\windows\flutter\ephemeral\.plugin_symlinks\window_manager\windows\include" /Zi
/nologo /W4 /WX /diagnostics:column /Od /Ob0 /D WIN32 /D _WINDOWS /D _HAS_EXCEPTIONS=0 /D _DEBUG /D NOMINMAX /D UNICODE
/D _UNICODE /D "CMAKE_INTDIR=\"Debug\"" /D _UNICODE /D UNICODE /Gm- /EHsc /RTC1 /MDd /GS /fp:precise /Zc:wchar_t
/Zc:forScope /Zc:inline /GR /std:c++17 /Fo"client1.dir\Debug\\" /Fd"client1.dir\Debug\vc142.pdb" /Gd /TP /wd4100
/errorReport:queue "D:\Projects\client1\windows\flutter\generated_plugin_registrant.cc" "D:\Projects\client1\windows\runner\flutter_window.cpp"
[  +64 ms]   generated_plugin_registrant.cc
[ +800 ms]   flutter_window.cpp
[+1004 ms]   Generating Code...
[ +145 ms] ResourceCompile:
[   +1 ms]   All outputs are up-to-date.
[  +51 ms] Link:
[   +1 ms]   C:\Program Files (x86)\Microsoft Visual
Studio\2019\Enterprise\VC\Tools\MSVC\14.27.29110\bin\HostX64\x64\link.exe /ERRORREPORT:QUEUE /OUT:"D:\Projects\client1\build\windows\runner\Debug\client1.exe" /INCREMENTAL /NOLOGO ..\flutter\Debug\flutter_wrapper_app.lib
..\plugins\flutter_acrylic\Debug\flutter_acrylic_plugin.lib
..\plugins\platform_device_id_windows\Debug\platform_device_id_windows_plugin.lib
..\plugins\url_launcher_windows\Debug\url_launcher_windows_plugin.lib
..\plugins\window_manager\Debug\window_manager_plugin.lib "D:\Projects\client1\windows\flutter\ephemeral\flutter_windows.dll.lib" kernel32.lib user32.lib gdi32.lib winspool.lib
shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib /MANIFEST /MANIFESTUAC:"level='asInvoker'
uiAccess='false'" /manifest:embed /manifestinput:"D:\Projects\client1\windows\runner\runner.exe.manifest"
/DEBUG /PDB:"D:/Projects/client1/build/windows/runner/Debug/client1.pdb" /SUBSYSTEM:WINDOWS /TLBID:1
/DYNAMICBASE /NXCOMPAT /IMPLIB:"D:/Projects/client1/build/windows/runner/Debug/client1.lib" /MACHINE:X64
/machine:x64 client1.dir\Debug\Runner.res
[   +5 ms]   client1.dir\Debug\flutter_window.obj
[   +1 ms]   client1.dir\Debug\main.obj
[        ]   client1.dir\Debug\utils.obj
[        ]   client1.dir\Debug\win32_window.obj
[        ]   client1.dir\Debug\generated_plugin_registrant.obj
[ +351 ms]   client1.vcxproj -> D:\Projects\client1\build\windows\runner\Debug\client1.exe
[   +7 ms] FinalizeBuildStatus:
[        ]   Deleting file "client1.dir\Debug\client1.tlog\unsuccessfulbuild".
[        ]   Touching "client1.dir\Debug\client1.tlog\client1.lastbuildstate".
[        ] Done Building Project "D:\Projects\client1\build\windows\runner\client1.vcxproj" (default
targets).
[   +6 ms] InitializeBuildStatus:
[        ]   Creating "x64\Debug\ALL_BUILD\ALL_BUILD.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[   +1 ms] CustomBuild:
[        ]   All outputs are up-to-date.
[   +6 ms] FinalizeBuildStatus:
[        ]   Deleting file "x64\Debug\ALL_BUILD\ALL_BUILD.tlog\unsuccessfulbuild".
[        ]   Touching "x64\Debug\ALL_BUILD\ALL_BUILD.tlog\ALL_BUILD.lastbuildstate".
[        ] Done Building Project "D:\Projects\client1\build\windows\ALL_BUILD.vcxproj" (default targets).
[   +5 ms] InitializeBuildStatus:
[        ]   Creating "x64\Debug\INSTALL\INSTALL.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[  +54 ms] PostBuildEvent:
[        ]   setlocal
[        ]   "C:\Program Files (x86)\Microsoft Visual
Studio\2019\Enterprise\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe" -DBUILD_TYPE=Debug -P
cmake_install.cmake
[        ]   if %errorlevel% neq 0 goto :cmEnd
[        ]   :cmEnd
[        ]   endlocal & call :cmErrorLevel %errorlevel% & goto :cmDone
[        ]   :cmErrorLevel
[        ]   exit /b %1
[        ]   :cmDone
[        ]   if %errorlevel% neq 0 goto :VCEnd
[        ]   :VCEnd
[ +121 ms]   -- Install configuration: "Debug"
[  +34 ms]   -- Up-to-date: D:/Projects/client1/build/windows/runner/Debug/data/icudtl.dat
[   +1 ms]   -- Up-to-date: D:/Projects/client1/build/windows/runner/Debug/flutter_windows.dll
[        ]   -- Up-to-date: D:/Projects/client1/build/windows/runner/Debug/flutter_acrylic_plugin.dll
[        ]   -- Up-to-date: D:/Projects/client1/build/windows/runner/Debug/platform_device_id_windows_plugin.dll
[   +1 ms]   -- Up-to-date: D:/Projects/client1/build/windows/runner/Debug/url_launcher_windows_plugin.dll
[        ]   -- Up-to-date: D:/Projects/client1/build/windows/runner/Debug/window_manager_plugin.dll
[  +21 ms]   -- Installing: D:/Projects/client1/build/windows/runner/Debug/data/flutter_assets
[   +1 ms]   -- Installing: D:/Projects/client1/build/windows/runner/Debug/data/flutter_assets/AssetManifest.json
[   +1 ms]   -- Installing: D:/Projects/client1/build/windows/runner/Debug/data/flutter_assets/assets
[        ]   -- Installing: D:/Projects/client1/build/windows/runner/Debug/data/flutter_assets/assets/fonts
[   +1 ms]   -- Installing: D:/Projects/client1/build/windows/runner/Debug/data/flutter_assets/assets/fonts/KhmerOS_sys.ttf
[   +2 ms]   -- Installing: D:/Projects/client1/build/windows/runner/Debug/data/flutter_assets/assets/i18n
[   +1 ms]   -- Installing: D:/Projects/client1/build/windows/runner/Debug/data/flutter_assets/assets/i18n/en.json
[   +2 ms]   -- Installing: D:/Projects/client1/build/windows/runner/Debug/data/flutter_assets/assets/images
[   +1 ms]   -- Installing: D:/Projects/client1/build/windows/runner/Debug/data/flutter_assets/assets/images/logo.png
[   +2 ms]   -- Installing: D:/Projects/client1/build/windows/runner/Debug/data/flutter_assets/FontManifest.json
[   +1 ms]   -- Installing: D:/Projects/client1/build/windows/runner/Debug/data/flutter_assets/fonts
[        ]   -- Installing: D:/Projects/client1/build/windows/runner/Debug/data/flutter_assets/fonts/MaterialIcons-Regular.otf
[   +8 ms]   -- Installing: D:/Projects/client1/build/windows/runner/Debug/data/flutter_assets/kernel_blob.bin
[ +210 ms]   -- Installing: D:/Projects/client1/build/windows/runner/Debug/data/flutter_assets/NOTICES
[   +2 ms]   -- Installing: D:/Projects/client1/build/windows/runner/Debug/data/flutter_assets/NOTICES.Z
[   +3 ms]   -- Installing: D:/Projects/client1/build/windows/runner/Debug/data/flutter_assets/packages
[        ]   -- Installing: D:/Projects/client1/build/windows/runner/Debug/data/flutter_assets/packages/fluent_ui
[   +1 ms]   -- Installing: D:/Projects/client1/build/windows/runner/Debug/data/flutter_assets/packages/fluent_ui/assets
[        ]   -- Installing: D:/Projects/client1/build/windows/runner/Debug/data/flutter_assets/packages/fluent_ui/assets/AcrylicNoise.png
[        ]   -- Installing: D:/Projects/client1/build/windows/runner/Debug/data/flutter_assets/packages/fluent_ui/fonts
[        ]   -- Installing: D:/Projects/client1/build/windows/runner/Debug/data/flutter_assets/packages/fluent_ui/fonts/FluentIcons.ttf
[   +1 ms]   -- Installing: D:/Projects/client1/build/windows/runner/Debug/data/flutter_assets/packages/window_manager
[        ]   -- Installing: D:/Projects/client1/build/windows/runner/Debug/data/flutter_assets/packages/window_manager/images
[   +1 ms]   -- Installing: D:/Projects/client1/build/windows/runner/Debug/data/flutter_assets/packages/window_manager/images/ic_chrome_close.png
[        ]   -- Installing: D:/Projects/client1/build/windows/runner/Debug/data/flutter_assets/packages/window_manager/images/ic_chrome_maximize.png
[        ]   -- Installing: D:/Projects/client1/build/windows/runner/Debug/data/flutter_assets/packages/window_manager/images/ic_chrome_minimize.png
[   +1 ms]   -- Installing: D:/Projects/client1/build/windows/runner/Debug/data/flutter_assets/packages/window_manager/images/ic_chrome_unmaximize.png
[  +13 ms] FinalizeBuildStatus:
[        ]   Deleting file "x64\Debug\INSTALL\INSTALL.tlog\unsuccessfulbuild".
[        ]   Touching "x64\Debug\INSTALL\INSTALL.tlog\INSTALL.lastbuildstate".
[        ] Done Building Project "D:\Projects\client1\build\windows\INSTALL.vcxproj" (default targets).
[        ] Build succeeded.
[        ]     0 Warning(s)
[        ]     0 Error(s)
[        ] Time Elapsed 00:00:29.80
[  +30 ms] Building Windows application... (completed in 30.6s)
[ +285 ms] Observatory URL on device: http://127.0.0.1:53639/wNE2qLFrNpE=/
[   +5 ms] Caching compiled dill
[  +67 ms] Connecting to service protocol: http://127.0.0.1:53639/wNE2qLFrNpE=/
[ +184 ms] Launching a Dart Developer Service (DDS) instance at http://127.0.0.1:0, connecting to VM service at
http://127.0.0.1:53639/wNE2qLFrNpE=/.
[  +97 ms] DDS is listening at http://127.0.0.1:53642/hioFPlMQMbA=/.
[  +60 ms] Successfully connected to service protocol: http://127.0.0.1:53639/wNE2qLFrNpE=/
[  +25 ms] DevFS: Creating new filesystem on the device (null)
[  +32 ms] DevFS: Created new filesystem on the device
(file:///C:/Users/leam_/AppData/Local/Temp/client175ca1f85/client1/)
[   +4 ms] Updating assets
[ +200 ms] Manifest contained wildcard assets. Inserting missing file into build graph to force rerun. for more
information see #56466.
[   +9 ms] Syncing files to device Windows...
[   +3 ms] <- reset
[        ] Compiling dart to kernel with 0 updated files
[   +1 ms] Processing bundle.
[   +1 ms] <- recompile file:///D:/Projects/client1/.dart_tool/flutter_build/generated_main.dart
d33b51ba-1de3-4308-91bd-5e7a962f4297
[        ] <- d33b51ba-1de3-4308-91bd-5e7a962f4297
[   +2 ms] Bundle processing done.
[ +224 ms] Updating files.
[        ] DevFS: Sync finished
[   +2 ms] Syncing files to device Windows... (completed in 235ms)
[   +1 ms] Synced 0.0MB.
[   +3 ms] <- accept
[   +3 ms] Connected to _flutterView/0x14dbc067ff0.
[   +4 ms] Flutter run key commands.
[   +2 ms] r Hot reload. 🔥🔥🔥
[   +1 ms] R Hot restart.
[        ] h List all available interactive commands.
[        ] d Detach (terminate "flutter run" but leave application running).
[        ] c Clear the screen
[   +1 ms] q Quit (terminate the application on the device).
[        ] 💪 Running with sound null safety 💪
[   +1 ms] An Observatory debugger and profiler on Windows is available at: http://127.0.0.1:53642/hioFPlMQMbA=/
[  +63 ms] The Flutter DevTools debugger and profiler on Windows is available at:
           http://127.0.0.1:9101?uri=http://127.0.0.1:53642/hioFPlMQMbA=/
[+17830 ms] Skipping target: gen_localizations
[   +6 ms] Skipping target: gen_dart_plugin_registrant
[   +1 ms] Skipping target: _composite
[   +1 ms] complete
[   +5 ms] Performing hot reload...
[ +122 ms] Scanned through 546 files in 99ms
[   +1 ms] Syncing files to device Windows...
[   +1 ms] Compiling dart to kernel with 0 updated files
[   +1 ms] Processing bundle.
[        ] <- recompile file:///D:/Projects/client1/.dart_tool/flutter_build/generated_main.dart
6d540f3e-ee99-4a35-8f4d-6fdd1400e84d
[   +1 ms] <- 6d540f3e-ee99-4a35-8f4d-6fdd1400e84d
[   +2 ms] Bundle processing done.
[  +32 ms] Updating files.
[  +20 ms] DevFS: Sync finished
[   +1 ms] Syncing files to device Windows... (completed in 60ms)
[   +1 ms] Synced 0.0MB.
[  +89 ms] Reassembling application
[ +201 ms] Hot reload performed in 458ms.
[   +6 ms] Performing hot reload... (completed in 483ms)
[   +2 ms] Reloaded 0 libraries in 511ms.
[+9363 ms] Skipping target: gen_localizations
[   +4 ms] Skipping target: gen_dart_plugin_registrant
[   +1 ms] Skipping target: _composite
[   +2 ms] complete
[   +6 ms] Performing hot restart...
[ +100 ms] Scanned through 546 files in 89ms
[   +2 ms] Syncing files to device Windows...
[   +1 ms] <- reset
[        ] Compiling dart to kernel with 0 updated files
[   +1 ms] Processing bundle.
[   +1 ms] <- recompile file:///D:/Projects/client1/.dart_tool/flutter_build/generated_main.dart
d8346d47-a8c7-44ef-8990-c36086297099
[   +1 ms] <- d8346d47-a8c7-44ef-8990-c36086297099
[   +2 ms] Bundle processing done.
[ +293 ms] Updating files.
[  +80 ms] DevFS: Sync finished
[   +1 ms] Syncing files to device Windows... (completed in 382ms)
[        ] Synced 39.6MB.
[   +1 ms] <- accept
[ +448 ms] Hot restart performed in 929ms.
[   +5 ms] Performing hot restart... (completed in 941ms)
[   +1 ms] Restarted application in 964ms.
[+51322 ms] Service protocol connection closed.
[   +2 ms] Lost connection to device.
[   +7 ms] DevFS: Deleting filesystem on the device (file:///C:/Users/leam_/AppData/Local/Temp/client175ca1f85/client1/)
[ +260 ms] Ignored error while cleaning up DevFS: TimeoutException after 0:00:00.250000: Future not completed
[  +46 ms] "flutter run" took 119,771ms.
[ +276 ms] ensureAnalyticsSent: 260ms
[   +4 ms] Running shutdown hooks
[   +1 ms] Shutdown hooks complete
[   +3 ms] exiting with code 0
```
</p>
</details>

<details><summary>Application Log on Event Viewer</summary>
<p>

```
The program client1.exe version 1.0.0.0 stopped interacting with Windows and was closed. To see if more information about the problem is available, check the problem history in the Security and Maintenance control panel.
 Process ID: 462c
 Start Time: 01d843ea32bd9df8
 Termination Time: 4294967295
 Application Path: D:\Projects\client1\build\windows\runner\Debug\client1.exe
 Report Id: d5261dc2-f059-4939-a7da-9cd676c2f3ee
 Faulting package full name:
 Faulting package-relative application ID:
 Hang type: Top level window is idle


------------------------------------------

Fault bucket 1433133757907078542, type 5
Event Name: AppHangB1
Response: Not available
Cab Id: 0

Problem signature:
P1: client1.exe
P2: 1.0.0.0
P3: 6243d506
P4: 5205
P5: 67246080
P6:
P7:
P8:
P9:
P10:

Attached files:
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.649b97ac-6cd4-41d4-9411-0b1745e6175c.tmp.WERInternalMetadata.xml

These files may be available here:
\\?\C:\ProgramData\Microsoft\Windows\WER\ReportArchive\Critical_client1.exe_aa9b36b97be7843b5801a6e62a11e8fc5eba4e_1befc861_2a27a37d-be58-4719-a6d8-daf72eff23c5

Analysis symbol:
Rechecking for solution: 0
Report Id: d5261dc2-f059-4939-a7da-9cd676c2f3ee
Report Status: 268435456
Hashed bucket: 3cfb70e1588fc4d723e3838f4bac0d8e
Cab Guid: 0
```
</p>
</details>

Update: I tried

- flutter upgrade --force
- flutter pub get
- flutter clean
- (delete windows directory on project which contain C++ files)
- flutter create --platforms=windows,macos,linux .

but these are not working.

**Comment by maheshmnj** [comment]
Hi @leamlidara, Thanks for filing the issue. I tried to reproduce this issue on windows by adding breakpoints in the default counter app, But I couldn't reproduce the app crash. Since you are seeing this issue frequently, can you please break down the reproducible steps for us to investigate this issue further?   

<details>
<summary>flutter doctor -v (windows)</summary>

```
[√] Flutter (Channel stable, 2.10.4, on Microsoft Windows [Version 10.0.19044.1586],
    locale en-US)
    • Flutter version 2.10.4 at C:\flutter_sdk\stable
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision c860cba910 (5 days ago), 2022-03-25 00:23:12 -0500
    • Engine revision 57d3bac3dd
    • Dart version 2.16.2
    • DevTools version 2.9.2

[√] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at C:\Users\mahesh\AppData\Local\Android\sdk
    • Platform android-31, build-tools 31.0.0
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop for Windows (Visual Studio Community 2019 16.11.4)
    • Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2019\Community
    • Visual Studio Community 2019 version 16.11.31727.386
    • Windows 10 SDK version 10.0.19041.0

[√] Android Studio (version 2020.3)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)

[√] VS Code (version 1.64.2)
    • VS Code at C:\Users\mahesh\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension can be installed from:
       https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter

[√] Connected device (3 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version
      10.0.19044.1586]
    • Chrome (web)      • chrome  • web-javascript • Google Chrome 99.0.4844.82      
    • Edge (web)        • edge    • web-javascript • Microsoft Edge 99.0.1150.30     

[√] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```
</details>

**Comment by leamlidara** [comment]
I'm sorry, but I unable to break it down, because it always lose connection on my device no matter I step down on breakpoint or not.

**Comment by maheshmnj** [comment]
Thanks for the update, However, I am unable to reproduce the issue. Labeling this issue for further investigation.
This issue looks similar to https://github.com/flutter/flutter/issues/100919

**Comment by leamlidara** [comment]
Thanks for your reply and stick with my issue. After your first reply, I decide to upgrade my Windows 10 SDK to your version 10.0.19041.0 and it is working perfectly fine.

**Comment by gspencergoog** [comment]
Thanks for the report!

(Desktop triage) closing as resolved, but if you discover reproduction case, please file a new issue.
